homeassistant/components/websocket_api/strings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "exceptions": {
+    "service_not_found": {
+      "message": "[%key:common::exceptions::service_not_found::message%]"
+    },
+    "service_child_not_found": {
+      "message": "Service {domain}.{service} not found."
+    }
+  }
+}
```

Comment:
These do not match what's sent by websocket_api

Reply:
They do afain.

Reply:
It seems there was a `__repr__` method that differed, hence the message set when initializing `ServiceNotFound` did not match since it was was overridden.

Reply:
Why is this marked as resolved?
These strings still do not match what's sent by websocket_api; websocket_api only uses translation_key `other_service_not_found`, but strings define two other keys: `service_not_found` and `service_child_not_found`.

Reply:
Sorry, I kept the original error text's. I have synced every thing now, also the `__repr__` variant and places the translation strings under the `homeassistant` integration. Also added placeholders.